### ISSUE TITLE
Find in Files does not search opened and changed files

### ISSUE THREAD

**Issue by david54321** [notepad-plus-plus/notepad-plus-plus#12424]
### Description of the Issue

Find in Files does not search in files that are open in NotePad++ but are changed elsewhere

### Steps to Reproduce the Issue

1. Open a text-file in NotePad++
2. Change the content of the file elsewhere (not in NotePad++)
3. Do NOT let Notepad++ reload the file
4. Use the Find in Files function to search for the new content just put inside the file
5. The file does not show up in the search results

### Expected Behavior

I expected that the search result would include the file

### Actual Behavior

The search result didn't find the file. Therefore, the search file-count was wrong too.

### Debug Information

Notepad++ v8.4.6   (64-bit)
Build time : Sep 25 2022 - 19:51:39
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 (64-bit) 
OS Version : 22H2
OS Build : 22621.755
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.8)
    NppConverter (4.4)
    NppExport (0.4)

**Comment by alankilborn** [comment]
For _Find in Files_ searches, if a file to be searched is already open in an editing tab in Notepad++, Notepad++ searches the in-memory tab buffer, not the file on disk.

If you think about it, this makes total sense because for Notepad++ to search a file it first has to load it.  If the file it is going to search is NOT open in a tab, Notepad++ first loads it into a "hidden virtual tab", then searches it and obtains results, then closes it.  If a file is already open in a tab (by a user having it open), Notepad++ skips the opening and closing steps.

The main idea is if a file is open in a tab, that contains the most-current data that a user is concerned with searching.

In short, this is "working as designed" and will not be changed or considered a defect.
This issue may be CLOSED.

**Comment by david54321** [comment]
I see. That is also what I thought about after posting the issue. It's by design - it makes good sense. Thanks for the quick response!
Regards
